### ISSUE TITLE
Bearer token input background color in send request modal

### ISSUE THREAD

**Issue by davidhu2000** [scalar/scalar#2878]
### What happens?

![Screenshot 2024-08-15 at 4 16 46 PM](https://github.com/user-attachments/assets/bae5fd78-0325-42fa-b67e-450fb40219e2)

Currently using the kepler theme. the background color is white in dark mode 

`@scalar/api-reference-react@0.3.74`

I am currently using 0.3.52, which has the same issue, i was able to work around it using custom css
```css
#http-bearer-token {
  background-color: transparent;
}
```

however, looks like in the latest version, the id is now includes a hash

```html
<input data-v-e4703b01="" placeholder="Token" id="http-bearer-token-xdl75QpcjH-WyuJ9UWrLc" autocomplete="off" class="border-none text-c-1 min-w-0 w-full peer px-2 py-1.5 outline-none" data-1p-ignore="" spellcheck="false" type="password" value="teststst">
```

### What did you expect to happen?

The background is the same color as the rest of the modal. 

### How can we reproduce the issue?

1. create a documentation using kepler theme
2. open the request client
3. see token input

**Comment by marclave** [comment]
hey @davidhu2000 , i wasnt able to see this with the kepler theme in latest version of @scalar/api-reference-react@0.3.80`

are you able to share a repo or a published docs site so we can look into this? :)

**Comment by davidhu2000** [comment]
@marclave i only saw the issue in the Bearer Token input. I'm not seeing Bearer token in your screenshot. Can you try using this API https://docs.scalar.com/project-default/version-default/reference-rs89lul6IYVhzkV1SCxLB#tag/authentication/GET/me

the docs unfortunately are not published with the updated version due to this styling issue. Let me know if what I posted above works, if not, i'll try to find some time to setup a repro

**Comment by davidhu2000** [comment]
I started an empty project and was unable to reproduce this issue. I took a deeper look and noticed that the bearer token input is `<input type='password' ... />`, which was affected to some default styling i have in my app. i was able to fix it by adding this custom css

```css
.scalar-client [type='password'] {
  background-color: transparent;
  color: #888;
  font-size: inherit;
}
```

This didn't affect other inputs such as cookies or headers because those aren't actually `<input/>` tags. Looks like they are using `<div/>`
